io/plugins/eu.esdihumboldt.hale.io.csv/src/eu/esdihumboldt/hale/io/csv/writer/AbstractTableInstanceWriter.java

```diff
@@ -225,20 +234,40 @@ private boolean shouldBeDisplayed(Object obj) {
 	}
 
 	private void addProperty(List<String> headerRow, Map<String, Object> row, Object property,
-			String propertyTypeName) {
+			String propertyTypeName, IOReporter reporter) {
 		if (!headerRow.contains(propertyTypeName)) {
 			headerRow.add(propertyTypeName);
 		}
+
+		if (property instanceof DefaultGeometryProperty<?>) {
```

Comment:
The `instanceof` check and the cast later should be done with the interface `GeometryProperty` and not the implementation class.

Reply:
@stempler the property that comes in the line 242 is a `GeometryProperty<?>` and not a `GeometryCollection`.

Reply:
I don't quite understand. This comment was about using `GeometryProperty` instead of `DefaultGeometryProperty`, which you also did, so what is the open point here from your view?